scripts/update_torch_wheels.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Must be YYYYMMDD format
+DATE=$1
+TORCH_WHEEL="gs://pytorch-tpu-releases/wheels/torch-1.1.0a0+stable-cp35-cp35m-linux_x86_64.whl"
```

Comment:
I'd isolate wheel file names into separate variables.

Reply:
Comments addressed in: https://github.com/pytorch/xla/pull/681